crates/contracts/src/alloy.rs

```diff
@@ -112,3 +112,45 @@ macro_rules! bindings {
         }
     };
 }
+
+crate::bindings!(GnosisSafe);
+crate::bindings!(GnosisSafeCompatibilityFallbackHandler);
+crate::bindings!(GnosisSafeProxy);
+crate::bindings!(GnosisSafeProxyFactory);
+
+pub mod macros {
+    #[macro_export]
+    macro_rules! tx_value {
+        ($call:expr, $value:expr) => {{
+            const NAME: &str = stringify!($call);
+            $call
+                .value($value)
+                .send()
+                .await
+                .expect(&format!("failed to send: {}", NAME))
+                .get_receipt()
```

Comment:
It seems like the receipt is not used, right?
```suggestion
                .watch()
```

Reply:
Yeah, I copied the logic from ethcontract, getting the receipt was the way to make it the most similar